closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -387,6 +397,11 @@ private static void checkIfKnownProfileSupported() {
                     + " on this platform.");
         }
 
+        if (profileID.contains("OpenJCEPlus") && !isOpenJCEPlusModuleExist) {
```

Comment:
The operands of `&&` should be reversed (doing the cheaper test first).

Reply:
Done.